Podfile.lock

```diff
@@ -3,7 +3,7 @@ PODS:
   - Alamofire (4.7.3)
   - AlamofireNetworkActivityIndicator (2.3.0):
     - Alamofire (~> 4.7)
-  - Automattic-Tracks-iOS (0.4.0):
+  - Automattic-Tracks-iOS (0.4.1):
```

Comment:
I don't understand these changes on Podfile.lock 🤔 
Maybe they are outdated on the release branch?

Reply:
Podfile.lock should be ok now!

Reply:
Looks great! 🎉